analyze_results.py

```diff
@@ -276,16 +276,16 @@ def __init__(self, file_path, ignored_issues):
 
         # Get the id of the resource, if any
         self.id = None
-        if file_type == "xml":
-            resource_tree = ET.parse(file_path)
-            try:
+        try:
+            if file_type == "xml":
+                resource_tree = ET.parse(file_path)
                 self.id = resource_tree.find(".//f:id", ns).attrib["value"]
-            except AttributeError:
-                self.id = None
-        elif file_type == "json":
-            resource_tree = json.load(open(file_path))
-            if "id" in resource_tree:
-                self.id = resource_tree["id"]
+            elif file_type == "json":
+                resource_tree = json.load(open(file_path))
+                if "id" in resource_tree:
+                    self.id = resource_tree["id"]
+        except:
+            pass    # In case of file not parseable, the error will be in the validation output file
```

Comment:
Hi @cdejonge , thanks for the PR. I just can't figure out what you mean with this comment, can you elaborate?